src/test/java/org/apache/commons/lang3/time/StopWatchTest.java

```diff
@@ -448,4 +473,570 @@ public void testToStringWithMessage() throws InterruptedException {
         final String splitStr = watch.toString();
         assertEquals(12 + MESSAGE.length() + 1, splitStr.length(), "Formatted split string not the correct length");
     }
+
+    @Nested
+    public class SupplierSupportRelatedTests {
```

Comment:
I do not think we need "Related" or "Support" in all these test class names.

A FooClassTest tests a FooClass. If you need something more precise in the naming, just say what the test does, "support" and "related" do not help understand what the test does IMO.

Reply:
I removed the suffix.